src/back/wgsl/writer.rs

```diff
@@ -1568,7 +1567,7 @@ impl<W: Write> Writer<W> {
                     Mf::Unpack2x16snorm => Function::Regular("unpack2x16snorm"),
                     Mf::Unpack2x16unorm => Function::Regular("unpack2x16unorm"),
                     Mf::Unpack2x16float => Function::Regular("unpack2x16float"),
-                    Mf::Inverse => {
+                    Mf::Inverse | Mf::Outer => {
```

Comment:
question: Is there a reason we don't remove `Inverse` and `Outer` from `MathFunction` altogether? I'm assuming it's "only" for the benefit of this diagnostic here, and the fact that other front ends might emit these.

Reply:
> I'm assuming it's "only" for the benefit of this diagnostic here, and the fact that other front ends might emit these.

Yeah, GLSL and SPIR-V have those so, ideally we'd polyfill them.